UltraStar Play/Assets/Scenes/SongEditor/Controls/SongMetaTags/BackgroundText.cs

```diff
@@ -0,0 +1,24 @@
+﻿using System;
+using System.Collections;
+using System.Collections.Generic;
+using System.Linq;
+using UnityEngine;
+using UnityEngine.UI;
+using UniInject;
+using UniRx;
+
+// Disable warning about fields that are never assigned, their values are injected.
+#pragma warning disable CS0649
+
+public class BackgroundText : AbstractSongMetaTagText
+{
+    protected override string GetSongMetaTagValue()
+    {
+        return songMeta.Background;
+    }
+
+    protected override string GetUiTextPrefix()
+    {
+        return "Background: ";
```

Comment:
what about translations? Shouldn't this text come from i18nManager.GetTranslation(I18NKeys.somekey)?

Reply:
Yes that would be the plan for another issue.

However, so far I did not care about i18n, except for the proof of concept and demo scene.
So at the moment, no text in the UI is translated (including Text components). It was no priority for me.